.github/workflows/check_msrv.yml

```diff
@@ -43,7 +43,24 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: check
-        args: --all --exclude=tracing-appender
+        args: --all --exclude=tracing-appender --exclude=tracing-serde
+
+  # TODO: remove this once tracing's MSRV is bumped.
+  check-msrv-serde:
```

Comment:
@hawkw let me know if you want me to roll this into the next item to save CI time. Technically, it does build just fine on 1.51 tho, if you want to keep those separate (probably doesn't matter for v0.2 release)

Reply:
(Either way, I should fix the copy/paste error in the comment)

Reply:
hmm, i would really prefer not to do a global MSRV bump, since `tracing`/`tracing-core` are dependencies of `tokio`, and i don't want to have to bump tokio's MSRV. having separate MSRVs for non-core crates is annoying, but it's probably preferable --- esp. since we would want to backport the `tracing-serde` changes to v0.1.x.

this is part of why i think it's probably a good idea to eventually move all of the non-core crates (such as `tracing-serde`) into their own repos...

Reply:
Yeah, to pull in `heapless`, we need 1.51. I could write my own collection types on top of a fixed size array (or use an older version of heapless I guess?) but it'd have relatively low value. You know your MSRV guarantees better than me tho.